test/cmd/templates.sh

```diff
@@ -11,11 +11,13 @@ os::log::install_errexit
 # This test validates template commands
 
 oc get templates
-oc create -f examples/sample-app/application-template-dockerbuild.json
+oc create -f examples/sample-app/application-template-dockerbuild.json 
+oc create -f examples/sample-app/application-template-dockerbuild.json -n openshift
```

Comment:
pretty sure that's not true... in test-cmd.sh:
```
  # switch back to a standard identity. This prevents individual tests from changing contexts and messing up other tests
  oc project ${CLUSTER_ADMIN_CONTEXT}
```

subtests start as cluster admin in an isolated project and can change users if they want